jcl/src/java.base/share/classes/com/ibm/tools/attach/target/Advertisement.java

```diff
@@ -109,7 +110,9 @@ private static StringBuilder createAdvertContent(String vmId, String displayName
 		addKeyValue(contentBuffer, KEY_VM_ID, vmId);
 		addKeyValue(contentBuffer, KEY_DISPLAY_NAME, (((null == displayName) || (displayName.length() == 0))? vmId: displayName));
 		addKeyValue(contentBuffer, KEY_NOTIFIER, CommonDirectory.MASTER_NOTIFIER);
-		
+		/*[IF Sidecar19-SE | Sidecar18-SE-OpenJ9]*/
```

Comment:
Why the limitation?
Also Sidecar19-SE is redundant as Java11 and later include Sidecar18-SE-OpenJ9

Reply:
Will fix.